### PR TITLE
fix(components): match Glimmer mobile styles to web

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

Because the Glimmer was using a `surface--background` token, it was actually quite difficult to parse when used overtop `surface--background` or other `surface--` variants.

In addition, it was inconsistent with the web Glimmer, which uses a neutral value.

### Changed

- semantic color tokens to rgb value so the Glimmer works better overlaid across assorted surfaces

### Fixed

- mobile Glimmer works better across various surface types

## Testing

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components-native/src/Glimmer/Glimmer.style.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const shineWidth = tokens["space-largest"];
 
 export const styles = StyleSheet.create({
   container: {
-    backgroundColor: tokens["color-surface--background"],
+    backgroundColor: "rgba(0, 0, 0, 0.05)",
```
Comment (chris-at-jobber) [inline packages/components-native/src/Glimmer/Glimmer.style.ts:8]: @MichaelParadis this is the same issue we have in web, where we'll probably need to bake in dark/light values within the component

File: packages/components-native/src/Glimmer/Glimmer.tsx [hunk 2]
```diff
@@ -83,15 +83,9 @@ export function Glimmer({
         <Svg>
           <Defs>
             <LinearGradient id="gradientShine" x1={0} y1={0.5} x2={1} y2={0.5}>
-              <Stop
-                offset="0%"
-                stopColor={tokens["color-surface--background"]}
-              />
+              <Stop offset="0%" stopColor="rgba(255,255,255,0)" />
```
Comment (cameron-jobber) [inline packages/components-native/src/Glimmer/Glimmer.tsx:86]: I think we might need to add `stopOpacity={0}`. This is what it looks like if I naively copy the code into `jobber-mobile`...

https://github.com/user-attachments/assets/be7ea3c8-35e3-4a7d-9e7e-4845453f6a32

and after I add the `stopOpacity`

https://github.com/user-attachments/assets/4abbda76-149c-420a-95ca-c0f2bad9bddf

I could also try it with a pre-release build to see if I mucked anything up with my copy pasta.

Comment (chris-at-jobber) [reply packages/components-native/src/Glimmer/Glimmer.tsx:86]: That certainly looks better!

Comment (chris-at-jobber) [reply packages/components-native/src/Glimmer/Glimmer.tsx]: @cameron-jobber pushed up the change

Comment (cameron-jobber) [reply packages/components-native/src/Glimmer/Glimmer.tsx]: This is what this looks like now in the pre-release build:

https://github.com/user-attachments/assets/afa554d4-92ac-4fa4-aa2d-c94fea0d94bc

👍 (IMO)

**Review by cameron-jobber — APPROVED** [review]
I'm not sure what additional review this needs, or by whom, but I'll give it a 👍.

File: packages/components-native/src/Glimmer/Glimmer.tsx (hunk 2 above)
Comment (jlelford) [reply packages/components-native/src/Glimmer/Glimmer.tsx]: This looks good to me @chris-at-jobber ! Thanks for the assist @cameron-jobber

Comment (cameron-jobber) [reply packages/components-native/src/Glimmer/Glimmer.tsx]: FWIW, [here is a PR](https://github.com/GetJobber/jobber-mobile/pull/9581) on `jobber-mobile` that installs the pre-release build. All tests are green. I've just kicked off a Bitrise build if that helps — I'm unsure of what the QA process is for Atlantis updates on JM.

Comment (chris-at-jobber) [reply packages/components-native/src/Glimmer/Glimmer.tsx]: @cameron-jobber I can take a quick QA pass on your build and then we'll get this merged!

**Pull request merged**